avaframe/com1DFA/timeDiscretizations.py

```diff
@@ -45,3 +45,21 @@ def getcflTimeStep(particles, dem, cfg):
 
     # return stable time step
     return dtStable
+
+
+def getMoussaTimeStep(dem, cfg):
```

Comment:
Set comment about naming. 
Improve the docstring. And there you could reference the article this is based on